src/RLCLiquidityUnifier.sol

```diff
@@ -65,7 +63,7 @@ contract RLCLiquidityUnifier is UUPSUpgradeable, AccessControlDefaultAdminRulesU
      * @param value The amount of RLC tokens to unlock and transfer
      */
     function crosschainMint(address to, uint256 value) external override onlyRole(TOKEN_BRIDGE_ROLE) {
-        RLC_TOKEN.safeTransfer(to, value);
+        RLC_TOKEN.transfer(to, value);
```

Comment:
The RLC contract is a controlled contract and its `transfer` and `transferFrom` functions are compatible with the ERC20 standard (return bool, correct events emitted, public visibility, ...) so we don't need to use `SafeERC20` here.

Reply:
How is it agnostic here!? In all cases we need the functions `transfer` and `transferFrom` in the target contract's interface.